app/src/main/java/com/connorlay/knitgrid/activities/PatternGridViewActivity.java

```diff
@@ -21,5 +26,17 @@ protected void onCreate(Bundle savedInstanceState) {
         transaction.replace(R.id.activity_pattern_grid_view_content_frame_layout, new PatternGridFragment());
         transaction.addToBackStack(null);
         transaction.commit();
+
+        if (isFirstLaunch()) {
```

Comment:
I dunno. Might be unnecessarily complicated, but we can decide. If eventually we have an explicit launch activity (say, a splash screen), we can do it there maybe?

What were you thinking that object would look like?